parsel/selector.py

```diff
@@ -175,8 +175,11 @@ def xpath(self, query):
             return self.selectorlist_cls([])
 
         try:
-            result = xpathev(query, namespaces=self.namespaces,
-                             smart_strings=self._lxml_smart_strings)
+            nsp = dict(self.namespaces)
+            nsp.update(kwargs.pop('namespaces', {}))
```

Comment:
To illustrate, with these changes

```
--- a/parsel/selector.py
+++ b/parsel/selector.py
@@ -161,7 +161,7 @@ class Selector(object):
     def _get_root(self, text, base_url=None):
         return create_root_node(text, self._parser, base_url=base_url)
 
-    def xpath(self, query, **kwargs):
+    def xpath(self, query, parsel_namespaces={}, **kwargs):
         """
         Find nodes matching the xpath ``query`` and return the result as a
         :class:`SelectorList` instance with all elements flattened. List
@@ -176,7 +176,7 @@ class Selector(object):
 
         try:
             nsp = dict(self.namespaces)
-            nsp.update(kwargs.pop('namespaces', {}))
+            nsp.update(parsel_namespaces)
             result = xpathev(query, namespaces=nsp,
                              smart_strings=self._lxml_smart_strings,
                              **kwargs)
```
the tests pass (with `parsel_namespaces`) but this changed test

```
         body = u"<p><input name='a' value='1'/><input name='b' value='2'/></p>"
         sel = self.sscls(text=body)
 
-        self.assertEqual([x.extract() for x in sel.xpath("//input[@value=$number]/@name", number=1)],
+        self.assertEqual([x.extract() for x in sel.xpath("//input[@value=$namespaces]/@name", namespaces=1)],
                          [u'a'])
```
fails with 
```
        try:
            nsp = dict(self.namespaces)
            nsp.update(parsel_namespaces)
            result = xpathev(query, namespaces=nsp,
                             smart_strings=self._lxml_smart_strings,
>                            **kwargs)
E                            TypeError: xpath() got multiple values for keyword argument 'namespaces'
```